runtime/doc/lua.txt

```diff
@@ -876,6 +876,32 @@ vim.region({bufnr}, {pos1}, {pos2}, {type}, {inclusive})       *vim.region()*
         whether the selection is inclusive or not, into a zero-indexed table 
         of linewise selections of the form `{linenr = {startcol, endcol}}` .
 
+                                             *vim.register_keystroke_callback()*
+vim.register_keystroke_callback({ns_id}, {fn})
```

Comment:
Using the terminology `ns_id` implies that `vim.api.nvim_buf_clear_namespace(bufnr, ns_id)` would clear the keystroke callbacks. Otherwise using ns_id here is incorrect, I think.

```
                Clears namespaced objects (highlights, extmarks, virtual text)
                from a region.
```

Instead with what you want, I would just take `{fn}` and return an `id` which you can use later to remove the callback. This can just be an incrementing integer since someone probably won't make 4 million callbacks.